src/main/java/org/folio/rest/transaction/AllOrNothingHandler.java

```diff
@@ -440,6 +458,30 @@ private Double getBudgetRemainingAmountForEncumbrance(Budget budget, String curr
     return result.getNumber().doubleValue();
   }
 
+  /**
+   * Calculates remaining amount for payment
+   * [remaining amount] = (allocated * allowableExpenditure) - (allocated - (unavailable + available)) - (awaitingPayment + expended)
```

Comment:
@craigmcnally Perhaps we should consider the encumbered value in this formula. For example, it is possible that we are trying to make a payment from a budget whose money is encumbered for orders not associated with this payment.